src/styles/block-card.css

```diff
@@ -0,0 +1,133 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__item-picture {
+  display: flex;
+  padding-bottom: 40px;
+  margin: 0 auto;
+}
+
+.card__imac-image {
+  width: 160px;
+  height: 134px;
+}
+
+.card__item-name {
+  color: var(--color-dark);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  padding-bottom: 4px;
+  margin: 0;
+}
+
+.card__item-code {
+  font-size: 10px;
+  line-height: 14px;
+  padding-bottom: 18px;
+  margin: 0;
+  color: var(--color-gray);
+}
+
+.card__item-rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 22px;
+}
+
+.stars {
```

Comment:
check all the previous comments before re-request, please
 > separate file for stars